controller/commentsController.js

```diff
@@ -51,3 +52,37 @@ exports.flagComment = async (req, res) => {
     errHandler(error, res);
   }
 };
+
+exports.updateComment = async (req, res, next) => {
+  const comment_id = req.params.commentId;
+  const commentBody = req.body.commentBody;
+  if (!commentBody || commentBody.trim() == "") {
+    return res.status(400).json({ message: "Invalid comment body" });
```

Comment:
Please use the responseHandler from the utils folder.

Reply:
Okay. I’ll send an update right away

Reply:
Okay, thanks.